FHEM/00_SIGNALduino.pm

```diff
@@ -167,7 +167,7 @@ my %matchListSIGNALduino = (
 			"4:OREGON"						=> "^(3[8-9A-F]|[4-6][0-9A-F]|7[0-8]).*",
 			"7:Hideki"						=> "^P12#75[A-F0-9]+",
 			"9:CUL_FHTTK"					=> "^T[A-F0-9]{8}",
-			"10:SD_WS07"					=> "^P7#[A-Fa-f0-9]{6}F[A-Fa-f0-9]{2}(#R[A-F0-9][A-F0-9]){0,1}\$",
+			"10:SD_WS07"					=> "^P7#[A-Fa-f0-9]{6}[AFaf][A-Fa-f0-9]{2}(#R[A-F0-9][A-F0-9]){0,1}\$",
```

Comment:
Brauchen wir hier eigentlich noch den zweiten Teil vom Regex (#R[A-F0-9][A-F0-9]){0,1}\$ ?

Reply:
Das ist die RSSI Übergabe die dort nicht übergeben wird oder?

Reply:
Ich vermute, das es der RSSI (gewesen) ist. Dieser Eintrag in %matchListSIGNALduino scheint aber sowieso keinen Einfluss zu haben. Zumindest funktionierten meine Tests vorher auch, obwohl nur 0xF an Position 7 erlaubt war.

Reply:
Ja, weil es in einer capture group gepackt wurde die 0 oder 1x vorkommen kann. 
Kann man somit auch einfach entfernen

Reply:
Habe ich hiermit getan.

Reply:
Das '\$' am Ende würde bedeuten, dass das Zeichen $ in der Regex gesucht wird.
Das wird bestimmt nicht vorkommen :(

'$' wäre das Ende der Zeichenkette und war bestimmt beabsichtigt